### PR TITLE
Added installation of Enterprise developer portal with Postgresql

### PR DESCRIPTION
**Pull request by Fadzli-fast**
## Description
Added installation of Enterprise developer portal with postgreSQL using separate docker containers and separate docker containers

## Motivation and Context
There is no documented way for customers to deploy enterprise portal if they plan on using postgres. 

## Preview Link
image
image
image
image

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [x] Something else (please add if needs merging to production or not)

## Checklist

- [x] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md [hunk 1]
```diff
@@ -215,6 +216,148 @@ or
 docker-compose down -v    # to shutdown the stack and remove the volume
 ```
 
+### Launch the Tyk Enterprise Developer portal with PostgreSql
```
Comment (slega) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md]: It should be 'PostgreSQL', not 'PostgreSql'.

**Review by slega — CHANGES_REQUESTED** [review]
Please replace enterprise PostgreSql with PostgreSQL. I don't see any issues other than that.

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md (hunk 1 above)
Comment (Fadzli-fast) [reply tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md]: Got it already made the changes

**Review by slega — APPROVED** [review]
LGTM

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md [hunk 2]
```diff
@@ -215,6 +216,148 @@ or
 docker-compose down -v    # to shutdown the stack and remove the volume
 ```
 
+### Launch the Tyk Enterprise Developer portal with PostgreSQL
+This guide demonstrates two approaches for setting up and running the portal with a Postgres database: using docker-compose and using separate Docker containers.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal.md]:
```suggestion
This guide demonstrates two approaches for setting up and running the portal with a [Postgres database]({{< ref "https://www.postgresql.org/" >}}): using docker-compose and using separate Docker containers.
```

**Pull request merged**